cli/pcluster/config_sanity.py

```diff
@@ -343,6 +343,11 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
             ):
                 self.__fail(resource_type, "DNS Hostnames not enabled in %s" % resource_value)
         # VPC Subnet Id
+        elif resource_type == "VPC_COHERENCE":
```

Comment:
I think you are missing the cfnconfig.py file in this PR

Reply:
done